src/main.js

```diff
@@ -383,60 +381,38 @@ function handleAppWebContentsCreated(dc, contents) {
 
   contents.on('will-navigate', (event, url) => {
     const contentID = event.sender.id;
-    const parsedUrl = new URL(url);
-    const urlIsTrusted = isTrustedURL(parsedUrl);
-    const urlIsTrustedExternalLoginPath = parsedUrl.hostname === customLogins[contentID].externalHostname;
-
-    // don't prevent custom login attempts (oath, saml)
-    if (!urlIsTrusted && !urlIsTrustedExternalLoginPath) {
-      event.preventDefault();
-
-      // reset custom login if in progress
-      if (customLogins[contentID].inProgress) {
-        customLogins[contentID].inProgress = false;
-        customLogins[contentID].externalHostname = null;
-
-        // redirect to starting url if set
-        if (customLogins[contentID].startingURL) {
-          event.sender.loadURL(customLogins[contentID].startingURL);
-          customLogins[contentID].startingURL = null;
-        }
-      }
+    const parsedURL = parseURL(url);
+    const urlIsTrusted = isTrustedURL(parsedURL);
+
+    if (urlIsTrusted) {
```

Comment:
```suggestion
    if (isTrustedURL(parsedURL)) {
```

Reply:
This suggestion can be applied in a bunch of places here. I think it's leftovers from when this was more complicated.